src/features/schema-manager/schema-details/showSchemaDetails.ts

```diff
@@ -21,6 +25,8 @@ function renderSchemaDetails({
 }) {
   const styleUri = getWebviewUri(webview, extensionUri, ['media', 'style.css'])
   const toolkitUri = getWebviewUri(webview, extensionUri, ['media', 'vendor', 'toolkit.js'])
+  const jsonSchema = await (await fetch(schema.jsonSchemaUrl)).json()
```

Comment:
We should fetch this outside of the `renderSchemaDetails` before we even create the panel, because otherwise user would see an empty panel before the file is actually loaded. This is my assumption, I might not be correct. But still, I don't think `renderSchemaDetails()` is responsible for fetching & parsing the json schema, we can just pass these as inputs (like we do with `schema`)

Reply:
moved

Reply:
moved